src/sidebar/components/ThreadCard.tsx

```diff
@@ -63,22 +61,30 @@ function ThreadCard({ frameSync, thread }: ThreadCardProps) {
   }, [focusRequest, store, thread.id]);
 
   return (
-    /* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
     <Card
       active={isHovered}
       classes="cursor-pointer focus-visible-ring theme-clean:border-none"
       data-testid="thread-card"
       elementRef={cardRef}
-      tabIndex={-1}
+      tabIndex={0}
+      role="button"
+      aria-label="Press Enter to scroll annotation into viewport"
       onClick={e => {
         // Prevent click events intended for another action from
         // triggering a page scroll.
-        if (!isFromButtonOrLink(e.target as Element) && thread.annotation) {
+        if (!isFromButtonOrLink(e.target as Element)) {
           scrollToAnnotation(thread.annotation);
         }
       }}
       onMouseEnter={() => setThreadHovered(thread.annotation ?? null)}
       onMouseLeave={() => setThreadHovered(null)}
+      onKeyDown={e => {
+        // Simulate default button behavior, where `Enter` and `Space` trigger
+        // click action
+        if (['Enter', ' '].includes(e.key)) {
+          scrollToAnnotation(thread.annotation);
```

Comment:
Are you certain here that `thread.annotation` exists?

Reply:
See https://github.com/hypothesis/client/pull/5539#discussion_r1228255682